### ISSUE TITLE
Name clash in emacs 29

### ISSUE THREAD

**Issue by benjamin-asdf** [magnars/string-edit.el#19]
There is a builtin package now called string-edit. 
It has some of the same functionality but does not cover the use case as well as magnars string-edit for me. 

I propose a rename, maybe to `string-edit-at-point`. 
The biggest issue on emacs 29 currently is that the minor mode name is the same, braking string-edit.

**Comment by magnars** [comment]
Agreed, it should probably have been `string-edit-at-point` from the beginning. Would you consider opening a PR for the change?

**Comment by benjamin-asdf** [comment]
@magnars Ok. One issue right now is that the readme says `package-install string-edit-at-point` but that is wrong.

**Comment by magnars** [comment]
Sounds like the most expedient fix is to correct the README now, and if someone wants to tackle the checkdoc and package-lint later, we can fix the melpa name then.
